chapters/0-introduction.tex

```diff
@@ -72,76 +194,128 @@ \section{Doing credible research at scale}
-
-We will not always give a lot of highly specific implementation details in this text,
-but will often point you to where they can be found on the \textbf{DIME Wiki}.\sidenote{Like this:
-\url{https://dimewiki.worldbank.org/Primary_Data_Collection}}
-The DIME Wiki is one of DIME Analytics' flagship products,
-a free online collection of our resources and best practices.\sidenote{
-\url{https://dimewiki.worldbank.org}}
-This book complements the DIME Wiki by providing a structured narrative
-of the data workflow for a typical research project.
-The Wiki, by contrast, provides unstructured but detailed information
-on how to complete each task, and links to further practical resources.
 
-\section{Adopting reproducible analytical practices through code}
+The Analytics team has invested many hours over many years
+in creating these data workflows for DIME team members to follow
+throughout the research process.
+It has also invested significant energy in the language and materials
+used to teach these workflows to new team members,
+and, in many cases, in software tools that support these workflows explicitly.
+This is because DIME team members often work on a wide range of projects
+with a wide range of teammates, and DIME has found
+that standardizing these core processes across all projects
+results in higher-quality work with fewer opportunities to make mistakes.
+In that way, the Analytics team is DIME's method of ``institutionalizing''
+a set of tools and practices, developed and refined over time,
+that give the unit a common base of knowledge and practice.
+This book is intended to share these ideas, practices, and tools
+with everyone who interacts with development data:
+graduate students, research assistants, policymakers, and empirical researchers.
 
-Modern quantitative research relies heavily on statistical software tools.
-Many development researchers come from economics and statistics backgrounds
+Whether it is collected through surveys, shared by partner organizations,
+or acquired from ``big'' data sources like sensors, satellites, or call data records,
+data handling and documentation is a key skill for researchers and staff.
+There are many places where standard processes and documentation practices
+are important to accurately convey and implement the intended research design in reality.\cite{vilhuber_lars_2020_3911311}
+For example, statistical code is typically an essential part of
+research design components such as sampling, randomization, and power analysis.
+As data is obtained, its quality must be validated,
+linkages between datasets must be organized and managed,
+and errors must be identified and corrected.
+Once raw data is in hand, researchers must create and analyze the final measures that
+are the motivation for the research study;
+then they must conduct the actual analyses.
+The exact details of each step of this work needs to be documented and made available
+alongside any final research outputs.
```

Comment:
I think the examples need work -- they show different points in the research cycle where documentation is needed but don't make clear why standardizing processes helps / is needed

Reply:
Tried more context in 2685c924c280c04c7e4703fefde1f716197ae553